aopy/analysis.py

```diff
@@ -147,10 +159,11 @@ def run_curvefitting(means, make_plot=True, fig_title='Tuning Curve', n_subplot_
         n_cols (int) : No. of cols in subplot
```

Comment:
There's no description for this function. Can you tell us what it does? Including that it calls three other functions